app/assets/stylesheets/application.scss

```diff
@@ -76,3 +75,17 @@
     margin: 0;
   }
 }
+
+#map-container {
+   height: 400px;
+   border-radius: 16px 16px;
+   border-color: #fff;
+   border-style: solid;
+   box-shadow: 2px 2px 10px #B1B1B1;
+   margin-top: 25px;
+   border-width: 7px;
+ }
+ #map-canvas {
```

Comment:
Avoid using id selectors<br>Line should be indented 0 spaces, but was indented 1 spaces